models/Media/Poll.js

```diff
@@ -66,14 +66,15 @@ const PollSchema = new Schema({
 PollSchema.statics.create = function (pollData, userId) {
   return new Promise((resolve, reject) => {
     const options = pollData.options.map(option => ({
-      text: option,
-      usersVoted: [],
+      text: option.text,
+      usersVoted: option.usersVoted || [],
       creator: ObjectId(userId),
     }));
 
     const newPoll = new this({
       title: pollData.title,
       multiSelect: pollData.multiSelect,
+      open: !!pollData.open,
```

Comment:
why the `!!`?

Reply:
In case open isn't present, it should convert it to false if it's null or undefined. Is it unneeded? I'm not sure

Reply:
huh smart, never thought to do that. Cool :+1: